acquire/tools/decrypter.py

```diff
@@ -244,10 +245,22 @@ def decrypt_header(header, fingerprint, key_file=None, key_server=None):
         return base64.b64decode(result["header"])
 
 
+def check_existing(in_path: Path, out_path: Path, status_queue: Queue) -> bool:
+    if out_path.exists():
+        _info(status_queue, f"Output file already exists: {out_path}")
+        return True
+
+    # Check if acquire file is compressed. If so, check if decompressed file already exists
+    if in_path.stem.endswith((".tgz", ".gz")) and (decompressed_file := out_path.with_suffix("")).exists():
```

Comment:
```suggestion
    if in_path.suffix in (".tgz", ".gz") and (decompressed_file := out_path.with_suffix("")).exists():
```

Reply:
I've been corrected offline, but can you add a comment with an example? E.g. 
`Path("file.tar.gz.enc").stem -> "file.tar.gz"`

Reply:
Added this to the existing comment 😄